src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-wed calendar--month-length-31">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--wed calendar--month-length--31">
```
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/7ef5f400-8e6c-45e1-86f8-130a558d5987)